conf/provisioning/apps/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# # config file version
+apiVersion: 1
+
+# apps:
```

Comment:
Thinking a few steps ahead I think I would like this to be part of provisioning plugins. Having both plugins and app provisioning seems slightly confusing to me.

Reply:
@bergquist Good point - I was also wondering if it shouldn't be for plugins, but I don't know the context.

Reply:
As described in https://github.com/grafana/grafana/issues/25526 
But if totally fine, not downloading plugins to start with.

Reply:
After discussion with Carl we think it make sense to rename the package apps to plugins, but still keep the object "apps" in that yaml file. Point was that data sources are provisioned separately currently and is also subject for installation in the future. It wouldn't make sense to have separate provisioning files for installation and configuration and we maybe want to change this in the future.